package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
```

Comment:
This package-lock.json looks broken, as it has not dependencies in it.

Reply:
fixed. I actually used `yarn install` :man_shrugging: